### PR TITLE
Add next Brno Pyvo

### PR DESCRIPTION
**Pull request by Dormouse759**

### CODE REVIEW

File: series/brno-pyvo/events/2019-02-28-testovaci.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+city: brno
+start: 2019-02-28 19:00:00
+name: Brněnské Pyvo
+topic: Testovací
+venue: artbar
+description: |
+    V únoru bude Pyvo na téma testování. Máte svöj osobní projekt, o kterém
```
Comment (anezkamll) [inline series/brno-pyvo/events/2019-02-28-testovaci.yaml]: svöj -> svůj

File: series/brno-pyvo/events/2019-02-28-testovaci.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    jeden konkrétní řádek, protože by se něco mohlo rozbít?
+    Podíváme se, jak si napsat testy tak, abychom tyto problémy nemuseli
+    (tolik) řešit.
+
+    ------
+
+    In February, there will be Pyvo on the testing topic. Do you have your own
+    project you are not sure if it does what it should do? Or are you not sure
+    whether you may change that one particular line because of fear of breaking
+    things? Let's have a look on how to write test not to worry about that
```
Comment (anezkamll) [inline series/brno-pyvo/events/2019-02-28-testovaci.yaml]: look on -> look at

File: series/brno-pyvo/events/2019-02-28-testovaci.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        při testování produktu oVirt (open-source virtualizační nástroj).
+
+        ------
+
+        Pytest is an open-source testing framework which is useful for a wide
+        variety of projects - be it unit tests for your personal program on
+        GitHub or a large testing set for a company product. No matter what
+        you need to test, thanks to Pytest you don't need to start from zero.
+        We will introduce you to our first tests in a few minutes. One of
+        the strongest parts of PyTest are it's extensions. At last we will
```
Comment (anezkamll) [inline series/brno-pyvo/events/2019-02-28-testovaci.yaml]: At last -> At last,

Comment (anezkamll) [reply series/brno-pyvo/events/2019-02-28-testovaci.yaml]: it's -> its
